_kbarticles/2024-12-06-migrating-opensearch.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Migrating to the OpenSearch Dashboard for Cloud.gov logs"
+date: December 6, 2024
+excerpt: Changes to expect in our logging system in December 2024
+
+---
+
+## What's Changing in December 2024
```

Comment:
Since the title of the post ("Migrating to the OpenSearch Dashboard for Cloud.gov log") renders as second-level heading, for semantics it could make sense to render these as third level headings and so on, but not a blocking suggestion